lib/ansible/modules/windows/win_psrepository.ps1

```diff
@@ -18,9 +18,14 @@ $installationpolicy = Get-AnsibleParam -obj $params -name "installation_policy"
 
 $result = @{"changed" = $false}
 
+$PackageProvider = Get-PackageProvider -ListAvailable | Where-Object { ($_.name -eq 'Nuget') -and ($_.version -ge "2.8.5.201") }
+if ($null -eq $PackageProvider) {
+    Find-PackageProvider -Name Nuget -ForceBootstrap -IncludeDependencies -Force | Out-Null
```

Comment:
Can we conditionally run this if required so we can set `changed = $true` if there was a change.

Reply:
Done.